src/value.rs

```diff
@@ -164,6 +165,7 @@ impl ToString for Value {
             Value::Num(ref x) => x.to_string(),
             Value::Bool(ref x) => x.to_string(),
             Value::Str(ref x) => x.to_owned(),
+            Value::Null => "null".to_owned(),
```

Comment:
Sorry, just thought about this last night.  Are we aligned with Shopify on this?

Does it matter?

Reply:
I am not sure, what we use the `ToString` for within templates. From the Shopify docs it is rather vague how this would come to be used. In their [cheat sheet they only explain `Nil` to be `falsy`](https://www.shopify.com/partners/shopify-cheat-sheet) and all other of their filters and loops seem to make a check for truthy-ness before executing, which probably stems from that being the implied case in ruby. And is already a behavior we don't have that way (which we discussed the other day).

I am also happy to make it `""`, which is probably closer to the interpretation of their underlying ruby implementation...

Reply:
You're link answered the question of what `ToString` should do

> Tags or outputs that return nil will not print anything to the page.

https://help.shopify.com/themes/liquid/basics/types#nil

btw if Shopify calls it `Nil` then we probably should too (`Value::Nil`).